osmoutils/accum/accum.go

```diff
@@ -239,6 +247,10 @@ func (accum AccumulatorObject) UpdatePositionCustomAcc(name string, numShares sd
 	return accum.AddToPositionCustomAcc(name, numShares, customAccumulatorValue)
 }
 
+func (accum AccumulatorObject) DeletePosition(name string) {
```

Comment:
We should unexpert this because it is unsafe to delete a position anywhere other than `ClaimRewards`

Reply:
agreed